index.html

```diff
@@ -36,13 +36,41 @@
         >
           <p
             id="cart-items-number"
-            class="text-orange text-[10px] font-manrope font-bold"
+            class="text-orange text-xs font-manrope font-bold"
           >
             3
           </p>
         </div>
       </button>
     </div>
+    <div
+      id="cart-modal"
+      class="bg-primary-transparent flex flex-col gap-3.5 absolute rounded-xl font-manrope p-6 z-80 top-14 right-10 pointer-events-none opacity-0 scale-0 transition-all duration-300"
```

Comment:
check z-80 as it is not a class accepted. See standarized z-index classes for tailwind https://tailwindcss.com/docs/z-index

Reply:
i removed the class because it wasn't necessary